tasmota/xnrg_15_teleinfo.ino

```diff
@@ -23,75 +23,107 @@
  * Teleinfo : French energy provider metering telemety data 
  * Source: http://hallard.me/category/tinfo/
  *
- * Hardware Serial will be selected if GPIO1 = [TELEINFO_RX]
+ * Denky ESP32 Teleinfo Template
+ * {"NAME":"Denky (Teleinfo)","GPIO":[65504,65504,65504,65504,65504,65504,65504,65504,65504,65504,65504,65504,65504,65504,65504,65504,0,65504,65504,65504,0,65504,65504,65504,0,0,0,0,65504,65504,65504,65504,65504,0,0,65504],"FLAG":0,"BASE":1}
```

Comment:
I just changed the ESP32 template by allowing 1 instead of 65504 which makes reading a lot better. I suggest you change it accordingly.

Reply:
Done